lib/order.rb

```diff
@@ -1,18 +1,52 @@
+require "csv"
+require "ap"
+
 module Grocery
   class Order
-    attr_reader :id, :products
+    attr_accessor :id, :products
```

Comment:
I'm curious why you chose to make these `attr_accessor`s instead of `attr_reader`s. As far as I can tell, you don't set these values anywhere outside this class, nor should you given the spec.

Leaving it an `attr_reader` is a safety measure, so that if you accidentally do something like `if order.id = 3` (using `=` instead of `==`), ruby will give you a big error instead of doing the wrong thing.